what4/src/What4/Protocol/Online.hs

```diff
@@ -236,6 +239,29 @@ checkSatisfiable proc rsn p =
         do assume conn p
            check proc rsn
 
+-- | Get `n` abducts from the SMT solver, the disjunction of which entail `t`, and bind them to `nm`
+getAbducts ::
+  SMTReadWriter solver =>
+  SolverProcess scope solver ->
+  Int ->
+  String ->
+  BoolExpr scope ->
+  IO [String]
```

Comment:
We don't have infrastructure to parse this result yet, and it would be a moderate lift to add it. I think it might be better to mark the API as unstable and note that it will change when we are able to add that infrastructure.